Makefile

```diff
@@ -66,7 +66,7 @@ else ifneq ($(OS), Darwin)
   else
     MIMALLOC_LIB = out/mimalloc/libmimalloc.a
     CPPFLAGS += -Ithird-party/mimalloc/include
-    LIBS += -Wl,-whole-archive $(MIMALLOC_LIB) -Wl,-no-whole-archive
+    LIBS += -Wl,-whole-archive $(MIMALLOC_LIB) -Wl,-no-whole-archive -lrt
```

Comment:
Don't you want to move this to line 29 instead of this line?

Reply:
Fair, wasn't sure where precisely. Thanks!